conf/local.conf

```diff
@@ -109,3 +109,9 @@ UEFI_SIGN_KEYDIR[vardepsexclude] += "TOPDIR"
 STM32_ROT_KEY_PATH ??= "${TOPDIR}/../tools/lmp-tools/security/stm32mp1/"
 STM32_ROT_KEY_PATH[vardepsexclude] += "TOPDIR"
 STM32_ROT_KEY_PASSWORD ??= "foundries"
+
+#
+# ComposeFS signatures
+#
+CFS_SIGN_KEYDIR ??= "${TOPDIR}/conf/keys/cfs"
+CFS_SIGN_KEYDIR[vardepsexclude] += "TOPDIR"
```

Comment:
after looking on the meta PR I see the CFS_SIGN_KEYNAME so it will be good to also add a default value
```
CFS_SIGN_KEYNAME ?= "cfs-dev"
```

Reply:
with that we can drop the ones one the meta PR and make it generic and not machine specific 
https://github.com/ldts/meta-lmp/commit/14ef7ef0e2afef8e512f479e971b23c57a657ad2#diff-444ca8738c9172ef4f06ce2d79dac863177b904d7f673d4bf8e292560a397c6fR132-R133